src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ProjectUpgradesPage/versions/upgradeForms/V3/UpgradeFundingCycle/hooks/LaunchFundingCycles.ts

```diff
@@ -33,7 +33,7 @@ export const useLaunchFundingCycles = ({
     const callbacks = {
       onDone() {
         console.info(
-          'Reconfigure transaction executed. Awaiting confirmation...',
+          'Cycle edit transaction executed. Awaiting confirmation...',
```

Comment:
lets just delete this console.info.

Reply:
Sure. Code could probably be simplified but I'll just delete that callback (both there and in `src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ReconfigureFundingCycleSettingsPage/hooks/reconfigureFundingCycle.ts`